Gemfile

```diff
@@ -103,6 +103,7 @@ end
 group :development, :test do
   gem "pry"
   gem "rspec-rails"
+  gem "rubocop", "~> 0.80.1", require: false
```

Comment:
Gonna keep the `require: false` because we don't need it for Rails server, console, etc.